lib/db.ts

```diff
@@ -7,18 +7,20 @@ declare global {
 let prisma: PrismaClient;
 
 async function connect() {
-    if (process.env.NODE_ENV === 'production') {
-        prisma = new PrismaClient();
-    } else {
-        if (!global.cachedPrisma) {
-        global.cachedPrisma = new PrismaClient();
+    if (typeof window === "undefined") {
```

Comment:
statement is needed as PrismaClient must be run in the server considering browser is not able to run it.